server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogs.java

```diff
@@ -199,6 +203,9 @@ public void collect(GCStatus status) {
       long removeStop = System.currentTimeMillis();
       log.info(String.format("%d total logs removed from %d servers in %.2f seconds", count,
           logsByServer.size(), (removeStop - logEntryScanStop) / 1000.));
+
+      count = removeFiles(recoveryLogs.values());
```

Comment:
Should this be `count += `?

Reply:
no, its used for the following log message about the number of recovery logs removed